### PR TITLE
Ti'rakqi Smuggler Ship Fixes

### PR DESCRIPTION
**Pull request by beepbopbeepbop**
As the title says. Approved by Unathi and Skrell Lore.

### CODE REVIEW

**Comment by beepbopbeepbop** [comment]
!review

File: maps/away/ships/tirakqi_smuggler/tirakqi_smuggler.dm [hunk 1]
```diff
@@ -5,7 +5,7 @@
 	prefix = "ships/tirakqi_smuggler/"
 	suffixes = list("tirakqi_smuggler.dmm")
 
-	sectors = list(SECTOR_TAU_CETI, SECTOR_ROMANOVICH, SECTOR_CORP_ZONE, SECTOR_VALLEY_HALE, SECTOR_BADLANDS)
+	sectors = list(SECTOR_TAU_CETI, SECTOR_ROMANOVICH, SECTOR_CORP_ZONE, SECTOR_VALLEY_HALE, SECTOR_BADLANDS, SECTOR_UUEOAESA)
```
Comment (DreamySkrell) [inline maps/away/ships/tirakqi_smuggler/tirakqi_smuggler.dm:8]: why not all of badlands at this point

Comment (beepbopbeepbop) [reply maps/away/ships/tirakqi_smuggler/tirakqi_smuggler.dm:8]: that should be fine to do i just didnt know that tag existed

File: maps/away/ships/tirakqi_smuggler/tirakqi_smuggler.dm [hunk 2]
```diff
@@ -5,7 +5,7 @@
 	prefix = "ships/tirakqi_smuggler/"
 	suffixes = list("tirakqi_smuggler.dmm")
 
-	sectors = list(SECTOR_TAU_CETI, SECTOR_ROMANOVICH, SECTOR_CORP_ZONE, SECTOR_VALLEY_HALE, SECTOR_BADLANDS)
+	sectors = list(ALL_BADLAND_SECTORS)
```
Comment (DreamySkrell) [inline maps/away/ships/tirakqi_smuggler/tirakqi_smuggler.dm:8]: this removes it from corporate sectors

set this to `sectors = list(ALL_TAU_CETI_SECTORS, ALL_BADLAND_SECTORS)`

**Comment by beepbopbeepbop** [comment]
i have just realised that the tirakqi ship has some missing mapping things that i will expand the scope of the pr to fit.

**Comment by beepbopbeepbop** [comment]
!wip

**Comment by beepbopbeepbop** [comment]
!review

**Comment by NonQueueingMatt** [comment]
fix the lint errors

**Review by Arrow768 — APPROVED** [review]

**Comment by DreamySkrell** [comment]
image
bad changelog

**Pull request merged**